client/packages/system/src/Asset/ListView/Toolbar.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import {
+  AppBarContentPortal,
+  Box,
+  FilterMenu,
+  useTranslation,
+} from '@openmsupply-client/common';
+
+export const Toolbar = () => {
+  const t = useTranslation('catalogue');
+
+  return (
+    <AppBarContentPortal
+      sx={{
+        paddingBottom: '16px',
+        flex: 1,
+        justifyContent: 'space-between',
+        display: 'flex',
+      }}
+    >
+      <Box display="flex" gap={1}>
+        <FilterMenu
+          filters={[
+            {
+              type: 'enum',
+              name: t('label.category'),
+              urlParameter: 'category',
+              options: [
+                {
+                  label: 'Insulated Containers',
+                  value: 'aaa-bbb-ccc',
```

Comment:
I assume you'll add queries to look up the proper values here? is that captured in another issue?

I'm getting a usablity issue, when I try to search in the category, type and class filters (I thought it would be an autocomplete, but it jumps to a new screen when I press `I`)
Has this been raised already?

![Kapture 2024-02-28 at 17 13 58](https://github.com/msupply-foundation/open-msupply/assets/8287373/528e2479-ae45-478b-b5e7-ee37be192dcd)

Reply:
it's not an autocomplete, just a SELECT. and yes, one of the testers raised the same issue - when the filters were created, the enum version uses select not autocomplete though